Command/DatabaseCreateCommand.php

```diff
@@ -54,27 +54,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $query  = 'CREATE DATABASE '. $dbName .';';
         }
 
-        try {
-            $manager = new ConnectionManagerSingle();
-            $manager->setConfiguration($this->getTemporaryConfiguration($config));
+        $manager = new ConnectionManagerSingle();
+        $manager->setConfiguration($this->getTemporaryConfiguration($config));
 
-            $serviceContainer = Propel::getServiceContainer();
-            $serviceContainer->setAdapterClass($connectionName, $config['adapter']);
-            $serviceContainer->setConnectionManager($connectionName, $manager);
+        $serviceContainer = Propel::getServiceContainer();
+        $serviceContainer->setAdapterClass($connectionName, $config['adapter']);
+        $serviceContainer->setConnectionManager($connectionName, $manager);
 
-            $connection = Propel::getConnection($connectionName);
+        $connection = Propel::getConnection($connectionName);
 
-            $statement = $connection->prepare($query);
-            $statement->execute();
+        $statement = $connection->prepare($query);
+        $statement->execute();
 
-            $output->writeln(sprintf('<info>Database <comment>%s</comment> has been created.</info>', $dbName));
-        } catch (\Exception $e) {
-            $this->writeSection($output, array(
-                '[Propel] Exception caught',
-                '',
-                $e->getMessage()
-            ), 'fg=white;bg=red');
```

Comment:
Instead of removing the whole try block, we should rather just re-throw the exception after writing to the output.